### ISSUE TITLE
Behaviors are not accessible through code

### ISSUE THREAD

**Issue by mradzinski** [roughike/BottomBar#741]
Let's say I'm using Anko or other DSL library to write my views through code. How can I set the behavior of the BottomBar? There should be a method to set the behaviour through code and it should be a priority.

**Comment by yombunker** [comment]
yeah currently there doesn't seem to be a way of doing this because it needs to know in advance those behaviors to know how to render things in the first place. This restriction might or not change in the next big release of the library when we are trying to add custom tabs/badges and things like that.
